lib/lunchbot_web/live/list_orders_live.ex

```diff
@@ -85,41 +21,40 @@ defmodule LunchbotWeb.ListOrdersLive do
     orders = Repo.all(Order, user_id: user_id)
     orders = Order |> Order.get_orders_by_user(user_id) |> Repo.all()
 
-    # turn the list of orders in a readable format to be given to the heex template
-    orders_heex =
-      Enum.into(orders, [], fn an_order -> transform(an_order) end)
-      |> IO.inspect(label: "List of user's previous orders:")
+    user_id_with_orders =
```

Comment:
Can we rebase and squash so we have the ecto associations commit first and then squash this fixup with the first MVP commit

Reply:
yup thats the plan once i clean the commits up